# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/server/devtools-server.js
# Commit: 895cf08432c4
# Full Hash: 895cf08432c40fe371ff83e846cb161430c6be4b
# Author: Nicolas Chevobbe <nchevobbe@mozilla.com>
# Date: 2020-10-27 04:41:26
# Regressor Bug: 1633712
# File Overlap Count: 1
# Description:
#   Bug 1633712 - [devtools] Create WorkerTargets as soon as possible. r=ochameau,devtools-backward-compat-reviewers.
#   
#   This patch adds support for dedicated worker targets in the Watcher actor.
#   Shared and Service workers are not handled yet.
#   
# ==============================================================================

diff -r 2ce79a9ed2d5 -r 895cf08432c4 devtools/server/devtools-server.js
--- a/devtools/server/devtools-server.js	Mon Oct 26 12:52:26 2020 +0000
+++ b/devtools/server/devtools-server.js	Mon Oct 26 12:53:43 2020 +0000
@@ -337,10 +337,10 @@
     return this._onConnection(transport, prefix, true);
   },
 
-  connectToParentWindowActor(devtoolsFrameActor, forwardingPrefix) {
+  connectToParentWindowActor(jsWindowChildActor, forwardingPrefix) {
     this._checkInit();
     const transport = new JsWindowActorTransport(
-      devtoolsFrameActor,
+      jsWindowChildActor,
       forwardingPrefix
     );
 